infrastructure/quick-deploy/aws/armonik.tf

```diff
@@ -76,4 +76,5 @@ module "armonik" {
     image = local.ecr_images["${var.pod_deletion_cost.image}:${try(coalesce(var.pod_deletion_cost.tag), "")}"].image
     tag   = local.ecr_images["${var.pod_deletion_cost.image}:${try(coalesce(var.pod_deletion_cost.tag), "")}"].tag
   })
+  depends_on = [module.ecr]
```

Comment:
Never put depends_on on modules. Why do you need the explicit dependency anyway?

Reply:
The deployment crashes when the pods in armoniK are tried to be created before the ecr creation and images copy are completed.